kaldi-native-fbank/csrc/kaldi-math.h

```diff
@@ -0,0 +1,43 @@
+// kaldi-native-fbank/csrc/feature-window.h
+//
+// Copyright (c)  2022  Xiaomi Corporation (authors: Fangjun Kuang)
+
+// This file is copied/modified from kaldi/src/feat/feature-window.h
+
+#pragma once
+
+#include <cmath>  // logf, sqrtf, cosf
+#include <cstdlib>  // RAND_MAX
+
+#ifndef M_PI
+#define M_PI 3.1415926535897932384626433832795
+#endif
+
+#ifndef M_2PI
+#define M_2PI 6.283185307179586476925286766559005
+#endif
+
+namespace knf {
+
+inline float Log(float x) { return logf(x); }
+
+// Returns a random integer between 0 and RAND_MAX, inclusive
+int Rand(struct RandomState* state = NULL);
```

Comment:
By the way, could you upload `kaldi-math.cc`? The `.cc` file contains the implementation for this function.

Reply:
strange, I did not port it, and yet it compiled locally...

Reply:
CI failed
https://github.com/csukuangfj/kaldi-native-fbank/actions/runs/8268120256/job/22640066618?pr=40#step:6:23
```
[ 40%] Building CXX object kaldi-native-fbank/csrc/CMakeFiles/test-online-fbank.dir/test-online-fbank.cc.o
[ 44%] Linking CXX executable ../../bin/test-online-fbank
/usr/bin/ld: ../../lib/libkaldi-native-fbank-core.so: undefined reference to `knf::Rand(knf::RandomState*)'
/usr/bin/ld: ../../lib/libkaldi-native-fbank-core.so: undefined reference to `knf::RandomState::RandomState()'
collect2: error: ld returned 1 exit status
make[2]: *** [kaldi-native-fbank/csrc/CMakeFiles/test-online-fbank.dir/build.make:98: bin/test-online-fbank] Error 1
make[1]: *** [CMakeFiles/Makefile2:369: kaldi-native-fbank/csrc/CMakeFiles/test-online-fbank.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 48%] Linking CXX shared library ../../../lib/libgtest.so
[ 48%] Built target gtest
make: *** [Makefile:146: all] Error 2
Error: Process completed with exit code 2.
```

By the way, you also need to add `kaldi-math.cc` to CMakeLists.txt